.github/workflows/cmake.yml

```diff
@@ -62,13 +62,14 @@ jobs:
       if: ${{ runner.os == 'Linux' }}
       uses: egor-tensin/setup-gcc@v1
       with:
-        version: 11
+        version: 9
 
     - name: Build
       run: |
         cmake -B build -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} ${{ matrix.extra-cmake-flags }}
         cmake --build build --config ${{ matrix.build-type }} --parallel
         cmake --install build --prefix build/install --config ${{ matrix.build-type }}
+        python3 -m pip wheel -v -w build/ .
```

Comment:
Wouldn’t this build everything twice?

Reply:
That's correct, yes I'm working on ensuring that the build of the Python wheel file works for all systems.

The next step I'd like to do is to get CMake to handle packaging the wheel so people can install these easily with pip from the distribution. Not as easily done with the cpython lib file. So it's a WIP and I'm aware of it :)